pkg/platform/model/vcs.go

```diff
@@ -485,31 +488,31 @@ func CommitPlatform(owner, prjName string, op Operation, name, version string, w
 	// version is not the value that AddCommit needs - platforms do not post a version
 	commit, fail := AddCommit(bCommitID, msg, op, NamespacePlatform(), platformID, "")
 	if fail != nil {
-		return fail
+		return fail.ToError()
```

Comment:
Can't flag the actual line since it didn't change, but you still have a failure on line 479